frontend/vite.config.js

```diff
@@ -5,12 +5,8 @@ import istanbul from 'vite-plugin-istanbul'
 
 const truthy = ['1', 'true']
 
-export default defineConfig(args => {
+export default defineConfig(() => {
   return {
-    build: {
```

Comment:
question (blocking): does this mean we aren't minifiying for build anymore?

Reply:
This build config was because we couln't actually use Vite in dev mode before and had to build in development mode whilst running locally. Vite will minify out of the box hen in prod mode.

However, from looking into this I've just noticed that we had mode set to development for our build command. So I've fixed that.